src/raftstore/store/peer.rs

```diff
@@ -427,6 +427,11 @@ impl Peer {
             .unwrap();
     }
 
+    pub fn signal_new_region(&self) {
+        self.coprocessor_host
```

Comment:
can it be wrapped in Peer::create() and Peer::replicate()?

Reply:
Good idea. Let me try

Reply:
how about merging it with register_delegates(), seem these two always call in same place

Reply:
It's a good idea, but we should give `register_delegates` a new name I think. Do you have any idea about that?